web/js/app.js

```diff
@@ -376,25 +389,285 @@ LR.utils.preSelectMultiSelects = function(annotationData, keys, form){
+            }
+        }
+        if(labelText){
+            let entity = {
+                "@context" : LR.CONTEXT,
+                "type" : "Person",
+                "name" : labelText
+            }
+            fetch(LR.URLS.CREATE, {
+                method: "POST",
+                mode: "cors",
+                headers: {
+                    'Content-Type': 'application/json;charset=utf-8'
+                },
+                body: JSON.stringify(entity)
+            })
+            .then(response => response.json())
+            .then(newEntity => {
+                let collectionAnno = {
+                    "@context" : LR.CONTEXT,
+                    "body": {targetCollection: collectionName},
+                    "target": newEntity["@id"],
+                    "creator": localStorage,
+                    "type": "Annotation"
+                }
+                fetch(LR.URLS.CREATE, {
+                    method: "POST",
+                    mode: "cors",
+                    headers: {
+                        'Content-Type': 'application/json;charset=utf-8'
+                    },
+                    body: JSON.stringify(collectionAnno)
+                })
+                .then(response => response.json())
+                .then(newAnno => {
+                    let tag = `<span class="tag is-small">${entity.name}</span>` 
+                    selectedTagsArea.innerHTML += tag
+                    let input = event.target.closest("input[type='hidden']")
+                    let delim = (input.hasAttribute("deer-array-delimeter")) ? input.getAttribute("deer-array-delimeter") : ","
+                    input.value += (delim+entity.name)
+                })
+                .catch(err =>{
+                    alert("There was a problem trying to create the Annotation that puts the entity into the collection.  Please check the network panel.")
+                    console.error("There was a problem trying to create the Annotation that puts the entity into the collection.  Please check the network panel.")
+                })
+            })
+            .catch(err =>{
+                alert("There was a problem trying to add the entity to the collection.  Please check the network panel.")
+                console.error("There was a problem trying to add the entity to the collection.  Please check the network panel.")
+            })
+        }
+        else{
+            alert("You must supply something as a label to add an entity to the collection.");
+            console.warn("You must supply something as a label to add an entity to the collection.")
+        }
+    }
+    else{
+        alert("The name of the collection to add to was not provided.  Check the button below and see if it knows the collection name or not.")
+        console.warn("The name of the collection to add to was not provided.  Check the button below and see if it knows the collection name or not.")
+        console.log(event.target)
```

Comment:
This is a bit much. You can `console.warn` with multiple arguments if you want it all logged.

Reply:
Yeah yeah yeah I need like an `LR.utils.warning()` logging system.